doc/rados/operations/control.rst

```diff
@@ -122,198 +133,227 @@ or delete them if they were just created. :
+weights have relative values that default to 1.00000; their values are not
+absolute, and these weights must be distinguished from CRUSH weights (which
+reflect the absolute capacity of a bucket, as measured in TiB). To reweight
+OSDs by utilization, run the following command:
 
 .. prompt:: bash $
 
    ceph osd reweight-by-utilization [threshold [max_change [max_osds]]] [--no-increasing]
 
-To limit the step by which any OSD's reweight will be changed, specify
-``max_change`` which defaults to 0.05.  To limit the number of OSDs that will
-be adjusted, specify ``max_osds`` as well; the default is 4.  Increasing these
-parameters can speed leveling of OSD utilization, at the potential cost of
-greater impact on client operations due to more data moving at once.
+By default, this command adjusts the override weight of OSDs that have ±20%
+of the average utilization, but you can specify a different percentage in the
+``threshold`` argument. 
+
+To limit the increment by which any OSD's reweight is to be changed, use the
+``max_change`` argument (default: 0.05). To limit the number of OSDs that are
+to be adjusted, use the ``max_osds`` argument (default: 4). Increasing these
+variables can accelerate the reweighting process, but perhaps at the cost of
+slower client operations (as a result of the increase in data movement).
 
-To determine which and how many PGs and OSDs will be affected by a given invocation
-you can test before executing. :
+You can test the ``osd reweight-by-utilization`` command before running it. To
+find out which and how many PGs and OSDs will be affected by a specific use of
+the ``osd reweight-by-utilization`` command, run the following command:
 
 .. prompt:: bash $
 
    ceph osd test-reweight-by-utilization [threshold [max_change max_osds]] [--no-increasing]
 
-Adding ``--no-increasing`` to either command prevents increasing any
-override weights that are currently < 1.00000.  This can be useful when
-you are balancing in a hurry to remedy ``full`` or ``nearful`` OSDs or
-when some OSDs are being evacuated or slowly brought into service.
+The ``--no-increasing`` option can be added to the ``reweight-by-utilization``
+and ``test-reweight-by-utilization`` commands in order to prevent any override
+weights that are currently less than 1.00000 from being increased. This option
+can be useful in certain circumstances: for example, when you are hastily
+balancing in order to remedy ``full`` or ``nearfull`` OSDs, or when there are
+OSDs being evacuated or slowly brought into service.
 
-Deployments utilizing Nautilus (or later revisions of Luminous and Mimic)
-that have no pre-Luminous clients may instead wish to instead enable the
-`balancer`` module for ``ceph-mgr``.
+Operators of deployments that utilize Nautilus (or later revisions of Luminous
+and Mimic) and that have no pre-Luminous clients might instead want to enable
+the `balancer`` module for ``ceph-mgr``.
```

Comment:
Nautilus or newer

likely instead want to enable

The `balancer` module does the work for you and achieves a more uniform result, shuffling less data along the way.  Note that when enabling the `balancer` module you will want to converge any changed override weights back to 1.00000 so that the balancer can do an optimal job.  If your cluster is very full, reverting these override weights before enabling the balancer may cause some OSDs to become full, so a phased approach may needed.

Reply:
Accepted and incorporated into the text with minor alterations.